lib/common/defaults.js

```diff
@@ -5,7 +5,8 @@ var Defaults = {};
 Defaults.DEFAULT_FEE_PER_KB = 10000;
 Defaults.MIN_FEE_PER_KB = 0;
 Defaults.MAX_FEE_PER_KB = 1000000;
-Defaults.MAX_TX_FEE = 1 * 1e8;
+Defaults.MIN_TX_FEE = 0;
+Defaults.MAX_TX_FEE = 0.1 * 1e8;
```

Comment:
I thought about this too. I think BWS should not exclude free transactions. Maybe in the future we can set a dynamic minimum based on the min_realy_fee and providing a flag to override the min?